### DISCUSSION TITLE
Netbox configuration using Secure LDAP LDAPS

### DISCUSSION THREAD

**Discussion by berkeleycowboy** [netbox-community/netbox#13877] [General]
Can anybody point me the direction of where I can find configuration necessary for Secure LDAP with Netbox.  

I'm currently utilizing a self-signed certificate.  I understand implementing TLS \ SSL will require me to purchase a certificate from a CA.  

How to you get Netbox \ nginx to utilize Secure LDAP once the certificate has been applied to the server ?  

Dennis

**Comment by jcollie** [comment]
Using "ldaps" in your config should be enough.
```
AUTH_LDAP_SERVER_URI = "ldaps://ldap.example.com"
```
